src/sign/index.ts

```diff
@@ -74,8 +78,16 @@ export default class Client {
     message.from = message.from ? getAddress(message.from) : checksumAddress;
     if (!message.timestamp)
       message.timestamp = parseInt((Date.now() / 1e3).toFixed());
-    const data: any = { domain, types, message };
-    const sig = await signer._signTypedData(domain, data.types, message);
+
+    const domainData = {
+      ...domain
+    };
```

Comment:
Isn't this the same?
```suggestion
    const domainData = domain;
```

Reply:
This will create a object reference and update `domain` when you update `domainData` 

Could confuse devs in future, if we add more params